src/growableHeap.js

```diff
@@ -6,7 +6,7 @@
 
 // Support for growable heap + pthreads, where the buffer may change, so JS views
 // must be updated.
-function GROWABLE_HEAP() {
+function maybeUpdateMemoryViews() {
```

Comment:
I think I prefer `checkMemGrowth` myself, since this function is only needed in the memory growth case.

Reply:
I mildly dislike it because "check" sounds synonymous with "assert" (check that some invariant holds) and doesn't convey that it actually is a side-effectful function that might update things, but I also don't want to bikeshed too long and happy to change if you disagree :)

Reply:
Hmm.. I see what you mean.  I would also accept `handleMemGrowth` but I think `checkMemGrowth` is slightly more meaningful to me.

Reply:
SGTM